cedar-policy/CHANGELOG.md

```diff
@@ -21,6 +24,8 @@ Cedar Language Version: TBD
 - Stabilized [RFC 82](https://github.com/cedar-policy/rfcs/pull/82), removing
   the experimental `entity-tags` feature flag. That functionality is now available
   without the feature flag.
+- Improved validation error messages around action ids (#1258, resolving
```

Comment:
I see it's actually listed under 4.1.0

Reply:
Good catch, didn't notice I had put it under 4.1... my fault.

@aaronjeline do you mind moving it?

Reply:
yup

Reply:
looks like it was a funny merge. I looked at your PR and it's in the right place.